src/bridge/SequencerInbox.sol

```diff
@@ -129,12 +130,13 @@ contract SequencerInbox is DelegateCallAware, GasRefundEnabled, ISequencerInbox
 
     /// @inheritdoc ISequencerInbox
     bytes1 public constant EIGENDA_MESSAGE_HEADER_FLAG = 0xed;
-    
+
     // gap used to ensure forward compatiblity with newly introduced storage variables
     // from upstream offchainlabs/nitro-contracts. Any newly introduced storage vars
     // made in subsequent releases should result in decrementing the gap counter
     uint256[38] internal __gap;
-    IRollupManager public eigenDARollupManager;
+    IEigenDACertVerifier public eigenDACertVerifier;
+    uint256 internal constant MAX_EIGENDA_CERTIFICATE_DRIFT = 100;
```

Comment:
we need to standardize on this name across integration. lets add it to spec?

Reply:
sounds good

Reply:
added link in spec https://www.notion.so/EigenDA-V2-Integration-Spec-12d13c11c3e0800e8968f31ef2c6a2b3?d=19c13c11c3e080789d99001c88cd090f&pvs=4#18813c11c3e080668187c5de4355cbeb
Now I'm not even sure we need this thing in op stack... can you check my comment there?